conf.d/go-metro.yaml.example

```diff
@@ -0,0 +1,39 @@
+# NOTE: for go metro to run unprivileged, you will have to set CAP_NET_RAW capabilities on the binary:
+#        (debian based) $ sudo apt-get update libcap
+#        (redhat based) $ sudo yum install libcap
+#        (redhat based - alternatively) $ sudo yum install compat-libcap1
+#        $ sudo setcap cap_net_raw+ep /opt/datadog-agent/bin/go-metro
+#
+#        Also, please note that go-metro logs to its own file - found in /var/log/datadog/go-metro.log.
```

Comment:
Last thing we could mention here: `go-metro` does _not_ appear as a check on the agent's `info` page (IMO this is counter-intuitive for users used to regular checks/JMXFetch checks)